tests/testthat/test-convergence.R

```diff
@@ -92,3 +92,95 @@ test_that("convergence diagnostics throw correct errors", {
   expect_error(ess_quantile(mu, probs = 1.2), "'probs' must contain values between 0 and 1")
   expect_error(mcse_quantile(mu, probs = 1.2), "'probs' must contain values between 0 and 1")
 })
+
```

Comment:
Since rstar is such a different diagnostic from the others, I think it makes sense to move the corresponding tests into it's own file.

Reply:
Done.